client/app/caseDistribution/components/StaticLever.jsx

```diff
@@ -25,7 +25,7 @@ const StaticLever = ({ lever }) => {
   const unit = valueArray[1];
 
   return (
-    <tbody>
+    <div>
```

Comment:
Should this just be a React.Element tag?
```suggestion
    <>
```